seq2science/rules/qc.smk

```diff
@@ -1,5 +1,7 @@
 import re
 
+from seq2science.util import sieve_bam
```

Comment:
I'm on my phone, but I think you also need to import "just" seq2science here, because seq2science.__version__ is used in the qc report